src/openhab.cpp

```diff
@@ -70,100 +66,161 @@ OpenHAB::OpenHAB(const QVariantMap& config, EntitiesInterface* entities, Notific
     } else if (!_url.endsWith('/')) {
         _url += "/";
     }
-    _sseReconnectTimer = new QTimer(this);
-
-
-
-    qCDebug(m_logCategory) << "setup";
+    context_openHab = this;
+    _sseNetworkManager = new QNetworkAccessManager(context_openHab);
+    _sseReconnectTimer = new QTimer(context_openHab);
+    _nam = new QNetworkAccessManager(context_openHab);
+    _nam = new QNetworkAccessManager(context_openHab);
```

Comment:
`_nam` doesn't need to be allocated twice.
There's also no need for dynamic allocation.

Reply:
done